CHANGELOG.md

```diff
@@ -14,7 +41,10 @@
 requirements
 ([#1128](https://github.com/walmartlabs/concord/pull/1128));
 - project: downgrade nimbus-jose-jwt
-([#1129](https://github.com/walmartlabs/concord/pull/1129)).
+([#1129](https://github.com/walmartlabs/concord/pull/1129));
+- project: enable full annotation
+processing
+([#1131](https://github.com/walmartlabs/concord/pull/1131)).
```

Comment:
`enable full annotation` is not int 2.25.1, right?

Reply:
My bad, thanks for noticing!